src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netfx;
```

Comment:
script doesn't work, it's broken :( 
you will have to manually make the change.